shared/js/wifi_helper.js

```diff
@@ -188,7 +188,10 @@ var WifiHelper = {
       // use ssid + security as a composited key
       var key = network.ssid + '+' +
         self.getSecurity(network).join('+');
-      networksObject[key] = network;
+      if (!networksObject[key] ||
```

Comment:
Please add a comment to tell others why we have to compare `relSignalStrength` here. This can make this part of code more readable.